src/common/templates/check-your-answers.njk

```diff
@@ -13,12 +13,44 @@
         {% endif %}
       </div>
     </dl>
+  
+    {% macro check_your_answers_row(label, value) %}
+      <div class="app-check-your-answers__contents">
+        <dt class="app-check-your-answers__question">{{ label }}</dt>
+        <dd class="app-check-your-answers__answer">{{ value }}</dd>
+      </div>
+    {% endmacro %}
+
+    {% macro check_your_answers_row_port_code(id, label, value) %}
+      {% if value =='ZZZZ' or value =='YYYY' %}
+        {# Hide the row #}
+      {% else %}
+        <div class="app-check-your-answers__contents">
+          <dt id="{{ id }}-label" class="app-check-your-answers__question">{{ label }}</dt>
+          <dd id="{{ id }}-value" class="app-check-your-answers__answer">{{ value }}</dd>
+        </div>
+      {% endif %}
+      
+    {% endmacro %}
+
+    {% macro check_your_answers_row_coordinates(id, label, latitude, longitude, __) %}
```

Comment:
Added id parameter to make E2E scripts easier, had to use xpath otherwise.

This should probably be applied to the normal row macro as well.